lib/components/TextEditor/components/HideContentInput/HideContentInput.tsx

```diff
@@ -31,7 +31,7 @@ function HideContentInput(props: HideContentInputProps) {
     )
   }
 
-  const [hideContent = [], setHideContent] = useLocalStorage<string[]>(
+  const [hideContent, setHideContent] = useLocalStorage<string[]>(
```

Comment:
in `useLocalStorage` there is a second argument that could resolve the issue with a new link for each renderer, but we don't need a default value